src/cljs/rems/collapsible.cljs

```diff
@@ -14,11 +15,14 @@
   (when callback
     (callback)))
 
-(defn- hide [id]
+(defn- hide [id callback]
   (.collapse (js/$ (str "#" id)) "hide")
   (.. (js/$ (str "." id "-less"))
       (collapse "hide")
-      (one "hidden.bs.collapse" (fn [_] (. (js/$ (str "." id "-more")) collapse "show"))))
+      (one "hidden.bs.collapse" (fn [_]
+                                  (. (js/$ (str "." id "-more")) collapse "show")
+                                  (when callback
+                                    (callback)))))
```

Comment:
this allows bootstrap element to communicate state back to reagent component so that UI can render "hidden/show" elements in correct order

Reply:
We may also want to consider a new implementation of the collapsible with Bootstrap 5 or so. Maybe CSS animations + React implementation is better.